src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -1258,6 +1271,102 @@ tr.status-warning td {
     margin: 0 1.5em 0 0;
 }
 
+#summary .terminal {
+    --header-status-request-method-color: var(--gray-400);
+    --header-metadata-key: var(--gray-400);
+    --header-metadata-value: var(--gray-300);
+
+    background: var(--terminal-bg);
+    border: solid var(--terminal-border-color);
+    border-width: 30px 4px 4px 4px;
+    border-radius: 3px 3px 0 0;
+    box-shadow: none;
+    color: var(--gray-100);
+    padding: 10px 15px;
+    position: relative;
+}
+#summary .terminal .status-request-method {
+    margin-right: 10px;
+}
+#summary .terminal.status-success .status-response-status-code {
+    margin-right: 1.5em;
+    text-transform: uppercase;
+}
+#summary .terminal.status-warning,
+#summary .terminal.status-warning .status-response-status-text {
+    color: var(--terminal-warning-color);
+}
+#summary .terminal.status-warning .status-response-status-code {
+    background: var(--terminal-warning-bg);
+    color: var(--black);
+    margin-right: 1.5em;
+    padding: 1px 5px;
+    text-transform: uppercase;
+}
+#summary .terminal.status-warning,
+#summary .terminal.status-warning .status-response-status-text {
+    color: var(--terminal-warning-color);
+}
+#summary .terminal.status-error,
+#summary .terminal.status-error .status-response-status-text {
+    color: var(--terminal-error-color);
+}
+#summary .terminal.status-error .status-response-status-code {
+    background: var(--terminal-error-bg);
+    color: var(--black);
+    margin-right: 1.5em;
+    padding: 1px 5px;
+    text-transform: uppercase;
+}
+#summary .terminal.status-error,
+#summary .terminal.status-error .status-response-status-text {
+    color: var(--terminal-error-color);
+}
+.macos #summary .terminal:before {
```

Comment:
As the display depend on the OS, could you provide screenshots of the 3 different renderings ?

Reply:
Good idea. I've updated the description to add that screenshot.